website/project/model.py

```diff
@@ -634,6 +636,8 @@ class Node(GuidStoredObject, AddonModelMixin):
 
     piwik_site_id = fields.StringField()
 
+    has_child_node_subscriptions = fields.ForeignField('subscription', list=True)
```

Comment:
@samanehsan Can we rename this to child_node_subscriptions, and can you add a comment above it describing how this field is?